### ISSUE TITLE
добавление обхода dpi по похожим символам в домене адреса

### ISSUE THREAD

**Issue by pipipupu-ui** [ValdikSS/GoodbyeDPI#720]
### I've made sure there's no existing feature request / Я убедился, что такой функциональности еще никто не предлагал

- [X] I've made sure there's no existing feature request / Я убедился, что такой функциональности еще никто не предлагал

### Describe your feature / Опишите ваше предложение

Начались проблемы с ютубом. Сама страница загружается, но видео не грузится.

У меня предположение, что это из-за какой-то проверки в домене адреса устройствами dpi. 
Я так думаю из-за того, что в консоле не приходят ответы от адресов типа "...googlevideo.com/..."
![гок](https://github.com/user-attachments/assets/284b530b-53dd-4f01-a1d3-1fb2164dacde)

Думаю нужно добавить проверку наличия данного словосочетания в домене для обхода блокировки такого типа, если это уже конечно нельзя сделать как-то по-другому.

версия "goodbyedpi-0.2.3rc3-2"
запускается "service_install_russia_blacklist_YOUTUBE"

**Comment by Amegatron** [comment]
В данном конкретном случае достаточно прописать домен googlevideo.com в списках доменов (хотя вроде бы он там есть уже). Было предложение раньше сделать шаблоны доменов по маске (типа google*.com, или более продвинуто через регулярки), но оно пока не зашло.

**Comment by pipipupu-ui** [comment]
> В данном конкретном случае достаточно прописать домен googlevideo.com в списках доменов (хотя вроде бы он там есть уже). Было предложение раньше сделать шаблоны доменов по маске (типа google*.com, или более продвинуто через регулярки), но оно пока не зашло. А понятие "похожих символов" в домене - очень растяжимое, если не сказать магическое (или даже мошенническое). Обычно на этом построены фишинговые сайты)

домен в списке есть, сама программа понимает что в например в `rr6---sn-n8v7znsy.googlevideo.com` присутствует  `googlevideo.com`?

**Comment by Amegatron** [comment]
> сама программа понимает что в например в rr6---sn-n8v7znsy.googlevideo.com присутствует googlevideo.com?

Да. Потому что та билиберда перед googlevideo.com - это поддомен.

**Comment by pipipupu-ui** [comment]
> > сама программа понимает что в например в rr6---sn-n8v7znsy.googlevideo.com присутствует googlevideo.com?
> 
> Да. Потому что та билиберда перед googlevideo.com - это поддомен.

ну ладно тогда, просто в списках есть домены по типу `yt3.ggpht.com`

**Comment by Myrakami** [comment]
Тоже самое. Ещё заметил что, если весь браузер закрыть с сохраненной вкладкой видео ютуба и снова открыть то, тогда видео иногда запускается и прогружается нормально.

**Comment by Myrakami** [comment]
Странно но 

> Такая же проблема с ютубом. Сам сайт открывает хорошо, но сами видео не грузит. Пробовал гудчеком подобрать, но ничего хорошего не выдало.

Странно но, после такой же проблемы отключил все расширения и начало грузить. Мб только у меня.

**Comment by LuckyCat14** [comment]
> Странно но
> 
> > Такая же проблема с ютубом. Сам сайт открывает хорошо, но сами видео не грузит. Пробовал гудчеком подобрать, но ничего хорошего не выдало.
> 
> Странно но, после такой же проблемы отключил все расширения и начало грузить. Мб только у меня.

нет, у меня тоже заработало после отключения расширений
спасибо за совет!

**Comment by Th3Ph0en1x** [comment]
Такая же проблема, сам ютуб нормально грузит, а видосы нет. Отключение расширений не помогает.

**Comment by Myrakami** [comment]
> Такая же проблема, сам ютуб нормально грузит, а видосы нет. Отключение расширений не помогает.

Попробовать тогда подобрать другие стратегии.

**Comment by Th3Ph0en1x** [comment]
> > Такая же проблема, сам ютуб нормально грузит, а видосы нет. Отключение расширений не помогает.
> 
> Попробовать тогда подобрать другие стратегии.

Стратегии?

**Comment by 0keu** [comment]
> > > Такая же проблема, сам ютуб нормально грузит, а видосы нет. Отключение расширений не помогает.
> > 
> > 
> > Попробовать тогда подобрать другие стратегии.
> 
> Стратегии?

он имеет ввиду подобрать другие аргументы, для этого можно попробовать GoodCheck (ищите на ntc.party) - она сама перепробует 253 разных комбинаций аргументов и покажет статистику досягаемости сайтов из списка блеклиста

**Comment by pipipupu-ui** [comment]
> > > > Такая же проблема, сам ютуб нормально грузит, а видов нет. Отключение расширений не помогает.
> > > 
> > > 
> > > Попробуй затем выбрать другие стратегии.
> > 
> > 
> > Стратегии?
> 
> он имеет значение, используя другие аргументы, для этого можно попробовать GoodCheck (ищите на ntc.party) - она ​​сама перепробует 253 разных комбинаций аргументов и покажет статистику досягаемости сайтов из списка блеклиста, для каждого алгоритма)

пробовал - не помогает, при тестах не может подключиться именно к `googlevideo.com`. Иногда к некоторым из таких может подключится, но не ко всем.

**Comment by 0keu** [comment]
> 
У меня так же, максимум получил 67/83 success, и из этого числа все фейлы это *googlevideo
и только 2 success по этим адресам.
Я поставил zapret от flowseal в пару к gdpi, ютуб и дискорд - полет нормальный

**Comment by SeregaMihalkin** [comment]
у меня тоже где с 18 октября перестал работать GoodbyeDPI, я скачал [zapret-discord-youtube](https://github.com/Flowseal/zapret-discord-youtube), пока работает. Автору GoodbyeDPI написал на почту 20 числа, чтобы обновил последнюю сборку, возможно не видел еще

**Comment by ValdikSS** [comment]
Как определить похожесть символов?
Добавление `googlevideo.com` в blacklist применяет методы разблокировки ко всем `*.googlevideo.com`, `*.*.googlevideo.com`, и т.п. Иными словами, ко всей зоне.
